Can not build on iOS

https://pipelines.actions.githubusercontent.com/serviceHosts/1990a263-8757-4bdb-9dd4-b3aaea312869/_apis/pipelines/1/runs/794/signedlogcontent/4?urlExpires=2023-06-09T13%3A48%3A08.6604963Z&urlSigningMethod=HMACV1&urlSignature=fGkYGY3ZFMpMBQciH9j6D%2Fm9jbaNTo97HkQyOHla%2Bm0%3D

This script exist in `simple_audio.podspec`:

```bash
mkdir Frameworks
cd Frameworks
if [ ! -d simple_audio.xcframework ]
then
  mkdir simple_audio.xcframework
  cd simple_audio.xcframework
  mkdir ios-arm64
  mkdir ios-arm64_x86_64-simulator
  curl -L "#{lib_url}/Info.plist?raw=true" -o Info.plist
  curl -L "#{lib_url}/ios-arm64/libsimple_audio.a?raw=true" -o ios-arm64/libsimple_audio.a
  curl -L "#{lib_url}/ios-arm64_x86_64-simulator/libsimple_audio.a?raw=true" -o ios-arm64_x86_64-simulator/libsimple_audio.a
fi
cd ../..
```

But directory `simple_audio.xcframework` already exists(in published package). So the following download script is never executed, making the compilation fail.

Another strange behavior is that even though I've removed the xcframework directory, it still can not build. It seems that the script is not executed?

Now the issue becomes:

```
Failed to build iOS app
Error (Xcode): Undefined symbol: _drop_opaque_Controls


Error (Xcode): Undefined symbol: _new_Controls


Error (Xcode): Undefined symbol: _new_box_autoadd_i64_0


Error (Xcode): Undefined symbol: _new_box_autoadd_metadata_0


Error (Xcode): Undefined symbol: _new_box_autoadd_player_0


Error (Xcode): Undefined symbol: _new_list_media_control_action_0


Error (Xcode): Undefined symbol: _share_opaque_Controls


Error (Xcode): Undefined symbol: _wire_callback_stream__static_method__Player


Error (Xcode): Undefined symbol: _wire_dispose__static_method__Player


Error (Xcode): Undefined symbol: _wire_get_progress__method__Player


Error (Xcode): Undefined symbol: _wire_has_preloaded__method__Player


Error (Xcode): Undefined symbol: _wire_is_playing__method__Player


Error (Xcode): Undefined symbol: _wire_loop_playback__method__Player


Error (Xcode): Undefined symbol: _wire_new__static_method__Player


Error (Xcode): Undefined symbol: _wire_normalize_volume__method__Player


Error (Xcode): Undefined symbol: _wire_open__method__Player


Error (Xcode): Undefined symbol: _wire_pause__method__Player


Error (Xcode): Undefined symbol: _wire_play__method__Player


Error (Xcode): Undefined symbol: _wire_play_preload__method__Player


Error (Xcode): Undefined symbol: _wire_playback_state_stream__static_method__Player


Error (Xcode): Undefined symbol: _wire_preload__method__Player


Error (Xcode): Undefined symbol: _wire_progress_state_stream__static_method__Player


Error (Xcode): Undefined symbol: _wire_seek__method__Player


Error (Xcode): Undefined symbol: _wire_set_metadata__method__Player


Error (Xcode): Undefined symbol: _wire_set_volume__method__Player


Error (Xcode): Undefined symbol: _wire_stop__method__Player


Error (Xcode): Linker command failed with exit code 1 (use -v to see invocation)


Encountered error while building for device.
```

Raw log:

```
           Undefined symbols for architecture arm64:
             "_drop_opaque_Controls", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_new_Controls", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_new_box_autoadd_i64_0", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_new_box_autoadd_metadata_0", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_new_box_autoadd_player_0", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_new_list_media_control_action_0", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_share_opaque_Controls", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_callback_stream__static_method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_dispose__static_method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_get_progress__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_has_preloaded__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_is_playing__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_loop_playback__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_new__static_method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_normalize_volume__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_open__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_pause__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_play__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_play_preload__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_playback_state_stream__static_method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_preload__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_progress_state_stream__static_method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_seek__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_set_metadata__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_set_volume__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
             "_wire_stop__method__Player", referenced from:
                 _$s12simple_audio22SwiftSimpleAudioPluginC5dummys5Int64VyFZ in simple_audio(SwiftSimpleAudioPlugin.o)
           ld: symbol(s) not found for architecture arm64
           clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I've confirmed that ios.zip is extracted. This log is generated after running `flutter clean`.

--- a/ios/simple_audio.podspec
+++ b/ios/simple_audio.podspec
@@ -8,7 +8,7 @@ cd Frameworks
 if [ ! -d ios.zip ]
 then
   curl -L "#{lib_url}" -o ios.zip
-  unzip ios.zip
+  unzip ios.zip -d 'simple_audio.xcframework'
 fi
 cd ..
 `

